src/components/WorkshopPage/WorkshopPage.js

```diff
@@ -148,76 +138,67 @@ const workshops = [
-				slides: 'https://docs.google.com/presentation/d/1kzZsZGdJ4h_GscDhjbLPs4L3j0XPIpE1-F_w2_zc6N4/edit?usp=sharing'
+				author: 'Lillian Gonick',
+				description: `Hello everyone! Welcome to the Intro to Git workshop README! In this workshop, we will
+				 be covering the version control system: Git. Together, we will walk through installing git, creating 
+				 our first repository, and merging two branches!`,
+				youtube: 'https://youtu.be/Gw2AGGJKnbg?si=0b5FhNxT__V_2Svk',
+				readme: 'https://github.com/uclaacm/hothXI-workshops/tree/main/intro-to-git',
+				slides: 'https://docs.google.com/presentation/d/1fU0VoWGm95IsHadN5STolL6PO74Et_sfQFq7WQ5_d24/edit?usp=drive_link'
 			},
 			{
 				title: 'Intro to Databases',
-				author: 'Satyen Subramaniam',
-				description: `All good projects need a strong foundation - a base for their data. Join us in this
-				workshop where we'll run through the use cases of databases, the structure of popular types of
-				databases, and how to choose a database. We'll also do a full demo using Google Firebase - a
-				beginner friendly non-relational database - and connect it to a custom website using React, all
-				completely from scratch! Get ready cus this workshop is gonna be absolutely fire (and make your
-				websites hella based) :)`,
-				youtube: 'https://youtu.be/l6D8X0L_L_g',
-				readme: 'https://github.com/uclaacm/hothX-workshops/tree/main/intro-to-databases',
-				slides: 'https://docs.google.com/presentation/d/1LE2CYSIvGKSbGdGuZuMC1hOnxjmKgP3uwH0-LPEWSFw/edit?usp=sharing'
+				author: 'Sneha Agarwal',
+				description: `Hi! Welcome to HOTHXI's Database workshop, in this workshop we'll be covering
+				the basics of what a database is and walkthrough how to set up, retrieve information, and
+				add information to our database! Before starting this workshop I recommend checking out 
+				the React JS or React Native workshops if you're not familar with them.`,
+				youtube: 'https://youtu.be/JZrRLf5Gmqg?si=BVWuiqDqKvcMYTxH',
+				readme: 'https://github.com/uclaacm/hothXI-workshops/tree/main/intro-to-databases',
+				slides: 'https://docs.google.com/presentation/d/1rkeQNLEeXlpSXsoHmFdx77FY4HgK-rTUSqRSs7JOe5E/edit?usp=drive_link'
+			},
+			{
+				title: 'Intro to Deep Learning',
+				author: 'Leon Lenk, Maxine Wu, & Jordan Lin (ACM AI)',
+				description: `Shoutout to the folks over at ACM AI for putting this together! They created this workshop
+				so serve as an introduction to deep learning, including a notebook linked on the readme to show you 
+				actual deep learning code and take you through a hands-on of how you can build this code yourself! 
+				In the process of building this model, they go over Big Data, avoiding bias and unintended patterns, 
+				understanding our data, making the network, and training`,
+				youtube: 'https://youtu.be/RggC7sb1_cE?si=4FLeRtfCLOgZezhU',
+				readme: 'https://bit.ly/acm_ai_24w_intro_dl',
+				slides: null
```

Comment:
In the deploy preview, there's still a button for slides, even though it's value is `null`.

Reply:
to fix this, i think we have to go into `Workshop.js` line [73](https://github.com/uclaacm/hoth.uclaacm.com/blob/9a9b64af26ad9c73fcd1d13caba1e2be18b2c8ed/src/components/WorkshopPage/Workshop.js#L73) and add a check if there's a null passed in like there is for the README value in line [65](https://github.com/uclaacm/hoth.uclaacm.com/blob/9a9b64af26ad9c73fcd1d13caba1e2be18b2c8ed/src/components/WorkshopPage/Workshop.js#L65)